Application/EdFi.Ods.AdminApi/E2E Tests/Multitenant/Admin API E2E 2.0 - Actions.postman_collection.json

```diff
@@ -0,0 +1,234 @@
+{
+	"info": {
+		"_postman_id": "0712f6e0-1d3d-42ef-96cb-cba0163e697b",
+		"name": "Admin API E2E 2.0 - Actions",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "26382147"
+	},
+	"item": [
+		{
+			"name": "v2",
+			"item": [
+				{
+					"name": "Actions",
+					"item": [
+						{
+							"name": "Authstrategies",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"pm.test(\"GET Actions: Status code is OK\", function () {\r",
+											"    pm.response.to.have.status(200);\r",
+											"});\r",
+											"\r",
+											"const response = pm.response.json();\r",
+											"\r",
+											"pm.test(\"GET Actions: Response matches success format\", function () {\r",
+											"    pm.expect(response[0]).to.have.property(\"id\");\r",
+											"    pm.expect(response[0]).to.have.property(\"name\");\r",
+											"    pm.expect(response[0]).to.have.property(\"uri\");\r",
+											"});\r",
+											"\r",
+											"const GetActionsSchema = {\r",
+											"  \"type\": \"array\",\r",
+											"  \"items\": [\r",
+											"    {\r",
+											"      \"type\": \"object\",\r",
+											"      \"properties\": {\r",
+											"        \"id\": {\r",
+											"          \"type\": \"integer\"\r",
+											"        },\r",
+											"        \"name\": {\r",
+											"          \"type\": \"string\"\r",
+											"        },\r",
+											"        \"uri\": {\r",
+											"          \"type\": \"string\"\r",
+											"        }\r",
+											"      },\r",
+											"      \"required\": [\r",
+											"        \"id\",\r",
+											"        \"name\",\r",
+											"        \"uri\"\r",
+											"      ]\r",
+											"    }\r",
+											"  ]\r",
+											"}\r",
+											"\r",
+											"pm.test(\"GET Actions: Validation Schema Response\", () => {\r",
+											"    pm.response.to.have.jsonSchema(GetActionsSchema);\r",
+											"});\r",
+											""
+										],
+										"type": "text/javascript"
+									}
+								},
+								{
+									"listen": "prerequest",
+									"script": {
+										"exec": [
+											""
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"method": "GET",
+								"header": [
```

Comment:
@simpat-adam 
Please set the tenant specific request header on collection level. So, that we can avoid updating individual endpoints.

Reply:
Adjusted.